dbms/src/Operators/AggregateContext.h

```diff
@@ -27,7 +27,10 @@ struct ThreadData
     size_t src_rows = 0;
     size_t src_bytes = 0;
 
-    Aggregator::AggProcessInfo agg_process_info{};
+    Aggregator::AggProcessInfo agg_process_info;
+    ThreadData(Aggregator * aggregator)
```

Comment:
how about just holding the `CancellationHoo`

Reply:
Because `CancellationHook` inside `Aggregator` maybe reset by `setCancellationHook`

Reply:
ok